### PR TITLE
Add decision about team structure

### PR DESCRIPTION
**Pull request by cammellos**
Added decision on team structure

### CODE REVIEW

**Comment by vkjr** [comment]
>Wallet (once the vertical team is created)

@cammellos, maybe a few words can be added about the wallet team, its difference, if it is "vertical"?

**Comment by cammellos** [comment]
> > Wallet (once the vertical team is created)
> 
> @cammellos, maybe a few words can be added about the wallet team, its difference, if it is "vertical"?

@vkjr should I create a separate doc  for that? seems like it could be a separate decision

File: doc/decisions/0014-team-structure.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+The people identified for each sections are as follow:
+
+- Messaging @flexsurfer
+- Onboarding @siddarthkay
+- Communities @ilmotta
+- Activity Center & Shell @J-Son89, followed by @Parveshdhull
+- Wallet (once the vertical team is created) @J-Son89, followed by @briansztamfater
+
+This will be a rotating post, every 4 months it will go to text next in line.
```
Comment (J-Son89) [inline doc/decisions/0014-team-structure.md]: typo -> it will go to the next in line

**Comment by vkjr** [comment]
> @vkjr should I create a separate doc for that? seems like it could be a separate decision

@cammellos, not sure) I perceive the wallet team as a part of the mobile team, so from this standpoint, it fits the document's topic - "team structure".  But if there are a lot of differences that need to be covered, maybe a separate decision doc is a better option.

**Review by siddarthkay — APPROVED** [review]

**Review by J-Son89 — APPROVED** [review]

**Review by vkjr — APPROVED** [review]

**Review by ilmotta — APPROVED** [review]

**Pull request merged**